postcss.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+	plugins: [
+		require('autoprefixer')({})
```

Comment:
Would that somehow share babel config for supported browsers? If not, we should probably add it at some point :)